tmpfiles.d/systemd-pstore.conf

```diff
@@ -0,0 +1,27 @@
+#
+#The systemd pstore service archives the contents of /sys/fs/pstore
```

Comment:
we usually place at least one space between `#` and the text.

`systemd pstore service` → `systemd-pstore.service(1)`

Reply:
done